sandbox/data-packages/coldp/1.1/version.json

```diff
@@ -0,0 +1,4 @@
+{
+  "version": "1.1",
+  "latestCompatibleVersion": "1.1"
```

Comment:
Can we put `1.0`? Wouldn't package `1.0` be a valid `1.1` package?

Reply:
yes it would! I just didn't properly understand what this is for ;)